addons/account_qr_code_emv/models/res_bank.py

```diff
@@ -2,14 +2,23 @@
 
 import re
 
-from odoo import _, api, models
+from odoo import _, api, fields, models
 
 from odoo.addons.account_qr_code_emv.const import CURRENCY_MAPPING
 
 
 class ResPartnerBank(models.Model):
     _inherit = 'res.partner.bank'
 
+    country_code = fields.Char(related='partner_id.country_code', string="Country Code")
+
+    @api.model
+    def _serialize(self, header, value):
+        if value is not None and value != '':
```

Comment:
```suggestion
        if value:
```
Do you need to check explicitly? Which use case would require this?

Reply:
the value can be 0 or False, so I guess it is better to check in this way